vendor/k8s.io/gengo/namer/namer.go.orig

```diff
@@ -1,394 +0,0 @@
-/*
```

Comment:
hmm, how is this file getting deleted, but our CI scripts also dont consider this as "modified" (im assuming that CI is also trying to delete it?)

Reply:
I've been confused about this file for a few weeks -- it seems like sometimes the `gen*.sh` scripts insist that it must exist, and other times they exist that it must not exist.  probably the second best thing (if we can't actually figure out what's going on) is to add it to .gitignore so that it stops toggling state and showing up as a diff.  I could add it in this PR if you think that's the right way out?